webui/sg-view/pages/index.vue

```diff
@@ -20,13 +20,22 @@
       get_index_by_name
   } from "../js/retrieval";
 
+  var host_history = process.env.HISTORY_URL;
+  if (host_history === "") {
+  if (process.client) {
+    var host_history = "http://"+window.location.hostname+":8001/history";
+    } else {
+      var host_history = "http://localhost:8001/history";
+    }
+  }
```

Comment:
I don't think the opening brace at 24 is closed (probably because line 25 needs to go?). I think `process.env.HISTORY_URL` is not set then it should be `undefined` or `null` so I would propose:
```
let host_history = process.env.HISTORY_URL;
if (!host_history) {
  host_history = "http://" + window.location.hostname + ":8001/history";
}
```
(Using `var` inside a block re-defines `host_history` so we need to avoid it inside the if-else; also, `let` is usually preferred nowadays).

Reply:
>   if (host_history === "") {
>   if (process.client) {
>     var host_history = "http://"+window.location.hostname+":8001/history";
>     } else {
>       var host_history = "http://localhost:8001/history";
>     }
>   }

line 24? It's closed at line 30? The code you referenced looks different from the lines quoted?

Reply:
Oh, I see; yeah, line 25 should be indented, right?

Reply:
> The code you referenced looks different from the lines quoted

I think `process.client` is always true with how this being run so I removed the `if-else`.